src/Symfony/Component/DependencyInjection/ContainerInterface.php

```diff
@@ -21,7 +22,7 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
  */
-interface ContainerInterface
+interface ContainerInterface extends PsrContainerInterface
```

Comment:
that basically means in whole framework you will not use `Psr` version, but `Symfony` version.
Then, one will be able to use concrete class that implements `Symfony` version as parameter for method expecting `Psr` version, but not the other way around. So, if I will have some package that follows `Psr`, I won't be able to use it directly in Symfony project, because everywhere the framework will require `Symfony` version instead